django/librarian/tasks.py

```diff
@@ -117,7 +117,10 @@ def process_document_helper(document, llm, force_azure=False):
                 "status_text": _("Adding to library..."),
             },
         )
+    print("---------------------Extracted text----------------")
+    print(document.extracted_text)
```

Comment:
Does printing document content to the logs have any implications in terms of data security, retention, etc.? Obviously there isn't really any increased risk, but I'm wondering if it might have "on-paper" implications for compliance.

Reply:
Will be removed in next push. Thx!